multinode-demo/remote_leader.sh

```diff
@@ -10,5 +10,5 @@ PATH="$HOME"/.cargo/bin:"$PATH"
 
 # Run setup
 USE_INSTALL=1 ./multinode-demo/setup.sh -p
-USE_INSTALL=1 SOLANA_CUDA=1 ./multinode-demo/leader.sh >leader.log 2>&1 &
 USE_INSTALL=1 ./multinode-demo/drone.sh >drone.log 2>&1 &
```

Comment:
@mvines Need to start drone before leader now. Do these changes look right? Anywhere else I need to change? Anyway to test this easily?

Reply:
lgtm, there may be another spot or two.  We can chase those down if needed after this lands